pkg/resourceinterpreter/defaultinterpreter/aggregatestatus.go

```diff
@@ -264,3 +265,50 @@ func aggregateStatefulSetStatus(object *unstructured.Unstructured, aggregatedSta
 
 	return helper.ToUnstructured(statefulSet)
 }
+
+func aggregatePodStatus(object *unstructured.Unstructured, aggregatedStatusItems []workv1alpha2.AggregatedStatusItem) (*unstructured.Unstructured, error) {
+	pod, err := helper.ConvertToPod(object)
+	if err != nil {
+		return nil, err
+	}
+
+	newStatus := &corev1.PodStatus{}
+	newStatus.ContainerStatuses = make([]corev1.ContainerStatus, 0)
+	readySum := 0
+	containerSum := 0
+	for _, item := range aggregatedStatusItems {
+		if item.Status == nil {
+			continue
+		}
+
+		temp := &corev1.PodStatus{}
+		if err = json.Unmarshal(item.Status.Raw, temp); err != nil {
+			return nil, err
+		}
+
+		for _, containerStatus := range temp.ContainerStatuses {
+			tempStatus := containerStatus
+			newStatus.ContainerStatuses = append(newStatus.ContainerStatuses, tempStatus)
+			if containerStatus.Ready {
```

Comment:
Here judgement can refer to: https://github.com/kubernetes/kubernetes/blob/8415ae647d2c433c89910a0e677094e3a20ffb2b/pkg/printers/internalversion/printers.go#L822-L824

Reply:
> Here judgement can refer to: https://github.com/kubernetes/kubernetes/blob/8415ae647d2c433c89910a0e677094e3a20ffb2b/pkg/printers/internalversion/printers.go#L822-L824

do you mean when container is ready,  then we can collect that containerStatus?

Reply:
The judgment here may not be strong enough. 

`container.Ready && container.State.Running != nil`

Reply:
> The judgment here may not be strong enough.
> 
> `container.Ready && container.State.Running != nil`

got it~